redis/src/main/scala/zio/redis/Output.scala

```diff
@@ -184,6 +184,21 @@ object Output {
       }
   }
 
+  case object OptionalLongOrLongArrayOutput extends Output[Either[Option[Long], Chunk[Long]]] {
```

Comment:
We introduced a convention recently to provide separate commands based on the output so that we don't force pattern match on the call site. Check out geo commands for an example.

Reply:
Yea I also thought about that but decided to keep it as close to the Redis API as possible. I'll the geo commands and split the LPOS into two commands.

Reply:
Done Split into `Lpos` and `LposCount` commands